test/solver.test.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+const chai = require("chai");
+require("mocha");
+const solver = require('../src/solver');
+
+describe("Sudoku one", function() {
+    it("should be solved", function() {
+        var unsolved_sudoku = "___26_7_168__7__9_19___45__82_1___4___46_29___5___3_28__93___74_4__5__367_3_18___";
```

Comment:
This is a small inconsistency. Using linter and formatter may help you with these.